vignettes/EpiNow2.Rmd

```diff
@@ -101,9 +101,9 @@ example_non_parametric_delay <- NonParametric(pmf = c(0.01, 0.1, 0.5, 0.3, 0.09)
 These distributions are passed to downstream functions in the same way that the 
 parametric distributions are. 
 
-Now, to the functions.
+Now, to the functionalities.
```

Comment:
As these are still functions it might be fine to keep this. The new wording feels a bit stilted to me.

Reply:
This has been reverted.